std/regex/internal/ir.d

```diff
@@ -49,10 +49,11 @@ CharMatcher[CodepointSet] matcherCache;
     }
 }
 
-@property ref wordMatcher()()
+static immutable CharMatcher wordMatcher;
+
+shared static this()
```

Comment:
This would add runtime to every D program, even if it's not using std.regex.

Reply:
From my measurements it's about 1ms and I think you still need to import std.regex to get it to run. If somebody has cheap lazy initialization for global immutables I'd gladly take it.

Reply:
Dmitry is right, that shouldn't be an issue.